xml/schema/ACL/ACL.xml

```diff
@@ -99,6 +105,10 @@
     <type>int unsigned</type>
     <comment>The ID of the object controlled by this ACL entry</comment>
     <add>1.6</add>
+    <pseudoconstant>
+      <callback>CRM_ACL_BAO_ACL::getObjectIdOptions</callback>
+      <prefetch>false</prefetch>
```

Comment:
@seamuslee001 I just noticed this. Why did you add this `<prefetch>` flag? I'm honestly not even sure what this would do when paired with a `<callback>`, I don' think that's ever been done before.